client/js/app/components/explorer/query_builder/extraction_properties_filter.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var ReactMultiSelect = require('../../common/react_multi_select.js');
+var ExplorerActions = require('../../../actions/ExplorerActions');
+
+var ExtractionPropertiesFilter = React.createClass({
+
+  _getKeys: function() {
+    var keys = _.keys(this.props.result);
+    var keyList = _.map(keys, function(key) {
```

Comment:
Also, we should probably upgrade this project to use `babel-latest` so we can just stay on top of the latest transpile options at all times.